style.css

```diff
@@ -359,3 +371,110 @@ h2 {
   justify-content: center;
   margin-left: 220px;
 }
+
+.briefings {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contact-container {
+  background-color: #fff;
+  width: 100%;
+  margin: 0;
+}
+
+.contact-me {
+  background-color: #6070ff;
+  width: 100%;
+  height: 100%;
+  margin: 0;
+  border-top-left-radius: 100px;
+  background-image: url("contact form shapes-mobile@2x.svg");
+  background-repeat: no-repeat;
+  background-position: right;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contact-title {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #fff;
+  margin-top: 100px;
+  margin-bottom: 0;
+}
+
+.contact-notice {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+  text-align: center;
+  margin-top: 12px;
+  width: 85%;
+}
+
+.contact-form {
+  width: 80%;
+  margin: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.input-setup {
+  width: 100%;
+  height: 50px;
+  border-radius: 5px;
+  border: 1px solid #ebebff;
+  padding: 0 10px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  margin-bottom: 20px;
+  color: black;
+}
+
+.text-area {
+  min-height: 200px;
+  max-height: 200px;
+  min-width: 100%;
+  line-height: 50px;
+  max-width: 100%;
+}
+
+.contact-button {
+  background: #fff;
+  border-radius: 8px;
+  border: 1px solid #fff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  width: 160px;
+  height: 48px;
+  text-align: center;
+  margin: 0;
+}
```

Comment:
Kindly fix this button to the left side as shown in the figma design below instead of the middle.

Your design             |  Figma design
:-------------------------:|:-------------------------:
![Screenshot from 2022-09-28 18-56-04](https://user-images.githubusercontent.com/65233860/192866448-327047dc-7a4e-49a6-9c28-3e60fb0f4291.png)  |  ![Screenshot from 2022-09-28 18-55-01](https://user-images.githubusercontent.com/65233860/192866487-7437cbba-5219-4f70-b70a-128e9ac81781.png)